lit_gpt/model.py

```diff
@@ -295,7 +298,9 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         # the intermediate size for fc_{1,2} is halved when compared to LLaMAMLP, thus implementing GeGLU
         x_fc_1 = self.fc_1(x)
         x_fc_2 = self.fc_2(x)
-        x = torch.nn.functional.gelu(x_fc_1, approximate="tanh") * x_fc_2
+        # x = torch.nn.functional.gelu(x_fc_1, approximate="tanh") * x_fc_2
+        # NOTE: in HF they don't use approximation
```

Comment:
also not in `gemma_pytorch`... so strange. clearly the keras and pytorch folks were different teams entirely lol

Reply:
Stranger things :)

Something is definitely odd with the google management, I would say.